pyseq.py

```diff
@@ -667,7 +667,7 @@ def extend(self, items):
                 raise SequenceError("Item (%s) is not a member of this "
                                     "sequence." % item)
 
-    def reIndex(self, offset, padding=None):
+    def reIndex(self, offset=0, padding=None, out_path=None):
```

Comment:
not a fan of the name of this variable. can we think of a better one? maybe dest?or dstpath?